common/app/model/dotcomrendering/MostPopular.scala

```diff
@@ -124,17 +123,17 @@ object OnwardItemNx2 {
     )
   }
 
-  def pressedContentToOnwardItemNx2(content: PressedContent)(implicit
+  def pressedContentToOnwardItem(content: PressedContent)(implicit
       request: RequestHeader,
-  ): OnwardItemNx2 = {
+  ): OnwardItem = {
 
     def pillarToString(pillar: Pillar): String = {
```

Comment:
i could be missing something, so feel free to ignore, is this doing the same thing as OnwardsUtils.correctPillar? maybe just the toLowerCase is different? i thought id mention in case there was an opportunity to reuse that function

Reply:
+1. I'd probably define correctPillar/determinePillar as part of the companion object for `OnwardItem` unless they're used outside that?

Reply:
We determined the best place for it to live is onwards utils as it is used outside the scope of onward item